examples/platforms/cc2538/alarm.c

```diff
@@ -32,9 +32,6 @@
  *
  */
 
-#include <stdbool.h>
```

Comment:
I might of got a little carried away with the find/replace... BTW, I think it would be better to make comments related to the std*.h file replacements in the PR specifically for that: #579.